index.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Website: Ademola Adebayo</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css" integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&family=Roboto:wght@100&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/yourcode.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <header class="main-head">
+    <nav class="toolbar">
+      <div class="top-toolbar">
+        <div><h2>Smart City</h2></div>
+        <a href=""><i class=" fas fa-bars fa-2x"></i></a>
+      </div>
+    </nav>  
+  </header>
+  <section class="headline">
+    <div class="section-head">
+      <h1>Ademola Adebayo</h1>
+      <h3>Software Developer</h3>
+      <div class="head-text">
+        <p>I can help you build a product,<br> feature or website Look through<br> some of my work and experience! If<br> you like what you see and have 
+        a<br> project you need coded, don’t<br> hestiate to contact me.</p>
+      </div>
+      <hr> 
+    </div>
+    <div class="lang-framework">
+      <div>
+        <p class="lang-col">Language</p>
+        <ul>
+          <li>Javascript</li>
+          <li>HTML</li>
+          <li>Ruby</li>
+          <li>CSS</li>  
+        </ul>
+      </div>
+      <div class="side-2">
```

Comment:
- [ ] [OPTIONAL] At the moment, there is a horizontal scroll on the page and it does not fit the viewport. Please remove the horizontal scroll as it does not create a good user experience.